### PR TITLE
chore: New line in perService config

### PR DESCRIPTION
**Pull request by FrankEssenberger**
Add a new line at the end of the per-service-config.json so that the prettier we provide does not change the file.

### CODE REVIEW

**Review by marikaner — CHANGES_REQUESTED** [review]
Forgot to send this out before...

File: packages/openapi-generator/src/generator.spec.ts [hunk 1]
```diff
@@ -159,10 +159,21 @@ describe('generator', () => {
                   \\"directoryName\\": \\"spec\\",
                   \\"serviceName\\": \\"spec\\"
                 }
-              }"
+              }
+              "
             `);
     });
 
+    it('adds a new line at the options', async () => {
```
[the remark this answers is not in the document]
Comment (FrankEssenberger) [reply packages/openapi-generator/src/generator.spec.ts:167]: you are right, that it is implicitly tested, but I prefer to have a separate test but I adjust it.

File: packages/util/src/string-formatter.ts [hunk 2]
```diff
@@ -61,3 +61,12 @@ export function pascalCase(str: string): string {
 export function kebabCase(str: string): string {
   return voca.kebabCase(str);
 }
+
+/**
+ * Convert a JSON object to a string using formatting in line with the prettier.
+ * @param json Object to be stringified
+ * @returns The JSON object as string.
+ */
+export function jsonStringify(json: any): string {
```
[the remark this answers is not in the document]
Comment (FrankEssenberger) [reply packages/util/src/string-formatter.ts]: renamed the function

**Review by marikaner — APPROVED** [review]
👍
I still have to ask for one thing: `jsonFileFormat` does not really follow our style guide (https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-verbs-in-function-names).
Maybe consider: `formatJson` ?

**Pull request merged**